lib/onelogin/ruby-saml/saml_message.rb

```diff
@@ -1,6 +1,6 @@
-require 'cgi'
-require 'zlib'
-require 'base64'
+require "cgi"
```

Comment:
i'd opt for the opposite here, based on hound/rubocop -- double quote `"` only when using string interpolation, and single quote `'` everywhere else